packages/nextjs/services/mongo/mongo.ts

```diff
@@ -0,0 +1,44 @@
+import getConfig from "next/config";
+import { User } from "../../models/user";
+import { InsertOneResult, MongoClient } from "mongodb";
+
+const { serverRuntimeConfig } = getConfig();
+const mongoConfig = serverRuntimeConfig.mongoConfig;
+const client = new MongoClient(mongoConfig.connectionStr);
+const usersCollection = client.db(mongoConfig.DB!).collection<User>("User");
```

Comment:
Probably a simple verification here that `connectionStr` and/or `DB` is not empty would be enough to throw an Error and know some of them are missed from the ENV.

And that would also remove the `!`

Reply:
agree, fixed